coveralls.json

```diff
@@ -10,3 +10,4 @@
     "test/factories"
   ]
 }
+
```

Comment:
Now it's one newline too many :joy:

Just `git checkout master coveralls.json` haha

Reply:
AHHHHHHHHHHH